README.md

```diff
@@ -112,16 +118,23 @@ an individual package directory.
 
 ## Building the SDK
 
+Once you have installed all the dependencies, you can build the entire SDK by
+running the following command the root of the SDK:
```

Comment:
at the root of the SDK.

I actually meant to create a new section 'Build the SDK'  immediately after 'Install Dependencies', because it's the natural flow. It's also good to have it before the Testing section, because testing requires building the projects first.

We probably don't need a new section, also it avoids having 2 'Build the SDK' sections. Can you please move this paragraph to the end of 'Install Dependencies' ?